### ISSUE TITLE
Replace Gson with Jackson

### ISSUE THREAD

**Issue by knoobie** [vaadin/form-filler-addon#30]
The title says it all. Flow is already using two Json libraries, adding a third (Gson) does not add any additional value. Please consider using the already in-use libraries.

**Comment by mgarciavaadin** [comment]
Thank you for your comment @knoobie, the project is in an early stage and it is really appreciated your feedback to improve the first release (coming soon)

**Comment by czp13** [comment]
Yes, thanks : D Just shoot your ideas for improvements :), really appreciated! 🙇

**Comment by knoobie** [comment]
No worries! I got curious from the flow feature flag PR ;)

**Comment by knoobie** [comment]
Closed by PR above :)
